Cordate.xcodeproj/project.pbxproj

```diff
@@ -575,7 +482,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Do we need to change this?

If we do, we need to update the podspec as well

Reply:
I guess we don't. For our internal dependencies we want iOS 12 and up but I'm not too sure about public ones like this.

Reply:
After some thought, makes sense to just lower the deployment target back down as others may have a lower deploy target